inc/woocommerce.php

```diff
@@ -16,7 +16,18 @@
  * @return void
  */
 function _s_woocommerce_setup() {
-	add_theme_support( 'woocommerce' );
+	add_theme_support( 'woocommerce', array(
+		'thumbnail_image_width' => 150,
+		'single_image_width'    => 300,
+		'product_grid'          => array(
+			'default_rows'    => 3,
+			'min_rows'        => 2,
+			'max_rows'        => 8,
+			'default_columns' => 4,
+			'min_columns'     => 2,
+			'max_columns'     => 5,
```

Comment:
```suggestion
			'max_columns'     => 6,
```
As we support 6 columns in the CSS.